corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -76,13 +84,29 @@ def _get_source_destination_indexes(self, adapter):
         return adapter.primary.index_name, adapter.secondary.index_name
 
     def _prepare_index_for_reindex(self, index_name):
+        es_manager.cluster_routing(enabled=True)
         es_manager.index_configure_for_reindex(index_name)
         es_manager.index_set_replicas(index_name, 0)
+        self._wait_for_index_to_get_healthy(index_name, sleep_time=5)
+        es_manager.cluster_routing(enabled=False)
+
+    def _wait_for_index_to_get_healthy(self, index_name, sleep_time=0):
+        for i in range(10):
+            health = es_manager.cluster_health(index=index_name)
+            status = health["status"]
+            if status == "green":
+                break
+
+            print(f"\tWaiting for index status to be green. Current status: '{status}'")
+            time.sleep(max(sleep_time, min(2 ** i, 30)))
```

Comment:
nit: Exponential backoff should make the `sleep_time` argument unnecessary. It could be removed, and this could be changed to `time.sleep(min(2 ** i, 30))`

Reply:
I was thinking of removing it first, but then I became lazy and went ahead with the suggestion 😅 
Would remove it right away.